ui/qt-lib/src/Ariadne/UI/Qt/Util.hs

```diff
@@ -0,0 +1,16 @@
+module Ariadne.UI.Qt.Util
+  ( formatBalance
+  , unitToHtml
+  ) where
+
+import Universum
+
+import Ariadne.UI.Qt.Face
+
+formatBalance :: Text -> UiCurrency -> Text
+formatBalance balance unit = balance <> " " <> unitToHtml unit
+
+unitToHtml :: UiCurrency -> Text
+unitToHtml ADA = "<img src=':/images/ada-symbol-big-dark.png'>"
+unitToHtml Lovelace = "Lovelace"
+
```

Comment:
AFAIK this empty line shouldn't exist :man_shrugging:

Reply:
Oops.